gateway-queue/src/lib.rs

```diff
@@ -154,13 +154,28 @@ impl Queue for LocalQueue {
     }
 }
 
+/// An implementation of [`Queue`] that will instantly allow any requests.
+///
+/// This can be used when running with a proxy gateway in between your
+/// client and the Discord gateway and should not be used in any other
+/// cases.
```

Comment:
Verbiage and clearer warning
```suggestion
/// An implementation of [`Queue`] that instantly allows requests.
///
/// Useful when running behind a proxy gateway. Running without a
/// functional queue **will** get you ratelimited.
````